src/include/storage/recovery/recovery_manager.h

```diff
@@ -154,6 +154,9 @@ class RecoveryManager : public common::DedicatedThreadOwner {
   // Number of recovered committed txns. Used for benchmarking
   uint32_t recovered_txns_;
 
+  // How often the recovery throughput metric is collected
+  const uint32_t recovery_metric_interval_ = 1;
```

Comment:
This should be `std::chrono::milliseconds`, and pass it in as a `RecoveryManager` parameter.